src/azure-cli/azure/cli/command_modules/find/custom.py

```diff
@@ -115,7 +115,7 @@ def call_aladdin_service(query):
     if telemetry_core.is_telemetry_enabled() and subscription_id is not None:
         context['subscriptionId'] = subscription_id
 
-    api_url = 'https://aladdindevwestus-app.azurewebsites.net/api/v1.0/examples'
+    api_url = 'https://app.aladdin.microsoft.com/api/v1.0/examples'
```

Comment:
Will this work in air-gapped cloud? Perhaps it should go here 

https://github.com/Azure/azure-cli/blob/4055456f27f756bebe6e042baaf2396eb4154e47/src/azure-cli-core/azure/cli/core/cloud.py#L358-L362

to make it customizable.

Reply:
this is not working on air-gapped clouds. In air-gapped cloud, the solution is offline pacakage now. Once the service available in air-gapped clouds, we'll need register it in cloud.py.